app/data_loading.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+import json
+
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+file_path = os.path.join(current_directory, "config.json")
+
+with open(file_path, "r") as file:
+    configs = json.load(file)
```

Comment:
Perform this operation in a function to prevent undesired code execution in some cases.